federated-mongodb/README.md

```diff
@@ -78,28 +94,29 @@ Now that the `kubefedctl` binary has been acquired the next step is joining the
 Using the `kubeconfig` file that was generated, verify the Operator has been successfully deployed.
 ~~~sh
 $ oc get csv -n federated-mongo
-NAME                DISPLAY      VERSION   REPLACES   PHASE
-federation.v0.0.10   Federation   0.0.10                Succeeded
+NAME                      DISPLAY            VERSION   REPLACES   PHASE
+kubefed-operator.v0.1.0   Kubefed Operator   0.1.0                Succeeded
 ~~~
+
 The next step is to federate the clusters using `kubefedctl`.
 ~~~sh
-kubefedctl join east1 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=federated-mongo
-kubefedctl join east2 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=federated-mongo
-kubefedctl join west2 --host-cluster-context east1 --add-to-registry --v=2 --federation-namespace=federated-mongo
+kubefedctl join east1 --cluster-context east1 --host-cluster-context east1 --v=2 --kubefed-namespace=federated-mongo
+kubefedctl join east2 --cluster-context east2 --host-cluster-context east1 --v=2 --kubefed-namespace=federated-mongo
+kubefedctl join west2 --cluster-context west2 --host-cluster-context east1 --v=2 --kubefed-namespace=federated-mongo
 
 for type in namespaces deployments.apps ingresses.extensions secrets serviceaccounts services persistentvolumeclaims configmaps
 do
-  kubefedctl enable $type --federation-namespace federated-mongo
+  kubefedctl enable $type --kubefed-namespace federated-mongo
 done
 ~~~
 
-Validate that the clusters are defined as `federatedclusters`.
+Validate that the clusters are defined as `kubefedclusters`.
```

Comment:
I believe this would be better placed above the types enablement, just after the join.